.github/api_update_reminder.md

```diff
@@ -0,0 +1,26 @@
+---
+title: A new release of the Shopify API is due soon
+assignees: andyw8 # @platform/dev-tools-education
+labels: automated
+---
+
+This is an automated reminder for the maintainers that a new Stable release of the Shopify API is due soon, so the library needs to be updated.
+
+A new library release should be prepared to:
+* add the upcoming Stable release
+* remove the oldest Stable release which will no longer be supported.
+
+The PR should be created as a **draft** but not yet be merged.
+
+The release schedule can be found at https://shopify.dev/concepts/about-apis/versioning
+
+Review the changelog and consider if anything in the library needs to change:
+
+https://shopify.dev/changelog
+
+Test against the upcoming release by using the Release Candidate.
+
+Another reminder issue will be created on the date of the next release.
```

Comment:
Is there any chance we can add a comment to the same issue?

Reply:
In theory, yes, but I think it might be a bit tricky to script...

Reply:
Nothing wrong with using a second issue then!